app/application.go

```diff
@@ -110,7 +110,11 @@ func (app *App) setupState(stateBytes []byte) error {
 
 	err = app.Context.govern.SetStakingOptions(initial.Governance.StakingOptions)
 	if err != nil {
-		return errors.Wrap(err, "Setup State")
+		return errors.Wrap(err, "Setup Staking Options")
+	}
+	err = app.Context.govern.SetEvidenceOptions(initial.Governance.EvidenceOptions)
```

Comment:
I think we should add withheight , just to be safe 
app.Context.govern.WithHeight(app.header.Height).SetEvidenceOptions(initial.Governance.EvidenceOptions)

Reply:
Than for the SetStakingOptions it also must be included, but I did not see an issue, when a self staking part were did.

Reply:
We have added a few changes for governance options to take height into consideration after self staking is implemented